lms/product/canvas/_plugin/grouping.py

```diff
@@ -47,8 +47,8 @@ def get_sections_for_grading(self, _svc, course, grading_student_id):
 
         return [sec for sec in course_sections if sec["id"] in learner_section_ids]
 
-    def get_group_sets(self, course_id):
-        return self._canvas_api.course_group_categories(course_id)
+    def get_group_sets(self, course):
+        return self._canvas_api.course_group_categories(self._custom_course_id(course))
```

Comment:
We already have the self._custom_course_id helper function here to deal with the difference in Canvas.